etc/project.conf.example

```diff
@@ -16,6 +16,8 @@ job_max_output = 100MiB
 # name of the build job task configuration file within the repo
 job_desc_file = kevinfile
 
+# git configuration
+depth_commits = 0
```

Comment:
perhaps `git_clone_depth` is a more speaking name for this?

Reply:
Yeah, it's good, but if I use fetch instead clone, `git_fetch_depth` is a good one?